tensorflow_datasets/video/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 """Video datasets."""
 
-from tensorflow_datasets.video.bair_robot_pushing import BairRobotPushingSmall
+from tensorflow_datasets.video.bair_robot_pushing import BairRobotPushing
```

Comment:
As @Conchylicultor pointed out, it would be best for now to only change the description so that it doesn't break the current users' code